docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.carousel.scss

```diff
@@ -0,0 +1,178 @@
+.hb-hero-overlay {
+  .paragraph--type--hs-carousel & {
```

Comment:
Good call on pulling these styles over to their own file. 👍

It looks like essentially all styles in this file are nested inside this `.paragraph--type--hs-carousel` class:

![image](https://user-images.githubusercontent.com/1256329/78397713-b428d680-75bf-11ea-97b6-57fa7bdfd673.png)

I think the code would be more understandable if we just brought it up to the top level like this:

```sass
.paragraph--type--hs-carousel {
  .hb-hero-overlay { ...
    ...
   
    &__image-wrapper { ...
    }

    &__text { ...
    }
  }
}

```

That way it matches the html structure, and doesn't require unnecessary `&`s. Thoughts?

Reply:
ooo smart

Reply:
yep doing this

Reply:
I saw the update and it definitely looks better, but I was picturing this going all the way to the top.

Like this:
```sass
.paragraph--type--hs-carousel {
  .hb-hero-overlay { ...
  }
}
```

Or maybe this (for less nesting):
```sass
.paragraph--type--hs-carousel .hb-hero-overlay {
  ...
}
```

Instead of this:
```sass
.hb-hero-overlay {
  .paragraph--type--hs-carousel & { ...
  }
}
```

Thoughts?

Reply:
oooo that makes sense

Reply:
I like the update, but I'm seeing a linting error in this file:

![image](https://user-images.githubusercontent.com/1256329/78598249-fd299680-781c-11ea-8dd2-f860e44f21c6.png)

I can push up a commit to fix this real quick.